graylog2-web-interface/src/components/indexers/IndexerClusterHealth.tsx

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+import React from 'react';
+import { useQueries } from 'react-query';
+
+import { Spinner } from 'components/common';
+import { Row, Col } from 'components/bootstrap';
+import { DocumentationLink, SmallSupportLink } from 'components/support';
+import DocsHelper from 'util/DocsHelper';
+import { IndexerClusterHealthSummary } from 'components/indexers';
+import { getIndexerClusterHealth, getIndexerClusterName } from 'components/indexers/api';
+import type FetchError from 'logic/errors/FetchError';
+import { GET_INDEXER_CLUSTER_HEALTH, GET_INDEXER_CLUSTER_NAME } from 'logic/reactQueryKeys';
+
+import IndexerClusterHealthError from './IndexerClusterHealthError';
+
+const useLoadHealthAndName = () => {
+  const [
+    { data: healthData, isFetching: healthIsFetching, error: healthError, isSuccess: healthIsSuccess },
+    { data: nameData, isFetching: nameIsFetching, error: nameError, isSuccess: nameIsSuccess },
+  ] = useQueries([
+    { queryKey: GET_INDEXER_CLUSTER_HEALTH, queryFn: getIndexerClusterHealth },
+    { queryKey: GET_INDEXER_CLUSTER_NAME, queryFn: getIndexerClusterName },
+  ]);
```

Comment:
Now where we use `react-query` let's define a refetch interval, like `{ refetchInterval: 5000, retry: 0 }`

This way we do not have multiple immediate retries, when the cluster is down and we are refetching the status every 5 seconds, which will update the status, when it changes while the page for the cluster health is open.